src/util/logger/Logger.cpp

```diff
@@ -11,7 +11,7 @@
 #include <iostream>
 #include <stdio.h>
 
-#if defined(WIN32) || defined(_WIN32) || defined(__WIN32__)
+#if defined(_WIN32) || defined(_WIN32) || defined(__WIN32__)
```

Comment:
`_WIN32` check is now duplicated

Reply:
The hole line is sort of inflated `#ifdef(_WIN32)` is sufficient. And I just replaced all WIN32 defines with _WIN32 for testing purposes.